src/js/providers/html5.js

```diff
@@ -340,7 +340,7 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
     function checkStartDateTime() {
         if (_videotag.getStartDate) {
             const startDate = _videotag.getStartDate();
-            const startDateTime = startDate.getTime();
+            const startDateTime = !isNaN(startDate) ? startDate.getTime() : NaN;
```

Comment:
instead of checking for `isNaN` how about checking if `getTime` is defined ? 
`const startDateTime = startDate.getTime  ? startDate.getTime() : NaN;`

Reply:
Much better. Will update.